proto/mixer.proto

```diff
@@ -526,6 +527,19 @@ service Mixer {
     };
   }
 
+  // Get event collection for {eventType, affectedPlaceDcid, date}.
+  // NOTE:
+  // - The affectedPlaceDcid is only for top-level places such as Earth,
+  // continents, and countries.
+  // - The date format should be: YYYY-MM.
+  rpc EventCollection(datacommons.v1.EventCollectionRequest)
+      returns (datacommons.v1.EventCollectionResponse) {
+    option (google.api.http) = {
+      get : "/v1/events"
```

Comment:
@beets @pradh Wonder if this is for general "Event" or for "Disaster/Climate Event"?